plugin/pkg/parse/host.go

```diff
@@ -24,11 +24,17 @@ func HostPortOrFile(s ...string) ([]string, error) {
 		if err != nil {
 			// Parse didn't work, it is not a addr:port combo
 			if net.ParseIP(host) == nil {
-				// Not an IP address.
-				ss, err := tryFile(host)
+				// Not an IPv4 address, for sure. Could be IPv6 with %zone?
+				addrs, err := net.LookupHost(host)
```

Comment:
you can't use `net.LookupHost` here because you may not have a resolver at your disposal

Reply:
I was afraid of that. I wasn't totally sure that this was the correct approach, but it seemed like a good first step.